internal/backends/qt/qt_window.rs

```diff
@@ -129,11 +128,27 @@ cpp! {{
             });
         }
 
+        /// If this window is a PopupWindow and the mouse event is outside of the popup, then adjust the event to map to the parent window
+        /// Returns the position and the rust_window to which we need to deliver the event
+        std::tuple<QPoint, void*> adjust_mouse_event_to_popup_parent(QMouseEvent *event) {
+            auto pos = event->pos();
+            if (auto p = dynamic_cast<const SlintWidget*>(parent()); p && !rect().contains(pos)) {
+                while (auto pp = dynamic_cast<const SlintWidget*>(p->parent())) {
+                    if (p->rect().contains(p->mapFromGlobal(event->globalPosition().toPoint())))
```

Comment:
QMouseEvent::globalPosition is new in Qt6. But QMouseEvent::globalPos is deprecated.
I guess i need to make a #ifdef dance to avoid the deprecated warning.